src/main/java/br/unb/cic/analysis/svfa/SVFAAnalysis.java

```diff
@@ -32,22 +34,23 @@ public String sootClassPath() {
     }
 
     @Override
-    public List<String> getIncludeList() {
+    public scala.collection.immutable.List<String> getIncludeList() {
```

Comment:
@rafaelmotaalves it's not clear what's the intention of the commit. whatever it is, this doesn't seem related to the intention. so better split the commit into two next time.